lib/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Great job so far @mwismann There are just a few changes I will suggest you implement and you'll be on your way to the next milestone :+1: 
- [ ] Kindly ensure that `Getters` and `Setters` for `@rentals` is present. Currently, you only have `getters` set for `:rentals` so kindly fix that :wink: